src/index.html

```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container stars--0">
```

Comment:
Fix it everywhere
<img width="863" alt="image" src="https://user-images.githubusercontent.com/99403198/211154735-1503db1e-50cc-47db-9c56-0354d6974943.png">

```suggestion
    <div class="stars stars--0">
```

Reply:
Didn't fix it